userapi/storage/tables/stats_table_test.go

```diff
@@ -79,6 +79,7 @@ func mustMakeAccountAndDevice(
 	accDB tables.AccountsTable,
 	devDB tables.DevicesTable,
 	localpart string,
+	serverName gomatrixserverlib.ServerName, // nolint:unparam
```

Comment:
Since we're using `serverName`, `nolint` can be removed?

Reply:
The `nolint` is because it always receives the same value and the linter is picky :-)